test/rubygems/test_gem_stream_ui.rb

```diff
@@ -5,7 +5,7 @@
 
 class TestGemStreamUI < Gem::TestCase
   # increase timeout with MJIT for --jit-wait testing
-  mjit_enabled = defined?(RubyVM::JIT) ? RubyVM::JIT.enabled? : defined?(RubyVM::MJIT) && RubyVM::MJIT.enabled?
+  mjit_enabled = defined?(RubyVM::MJIT) ? RubyVM::MJIT.enabled? : defined?(RubyVM::MJIT) && RubyVM::MJIT.enabled?
```

Comment:
This seems to test multiple times the same thing, i.e., it looks like it should just be:
```ruby
mjit_enabled = defined?(RubyVM::MJIT) && RubyVM::MJIT.enabled?
```

Reply:
Can you create a pull request?

Reply:
https://github.com/rubygems/rubygems/pull/5288